geoh5py/io/h5_writer.py

```diff
@@ -434,12 +457,76 @@ def write_color_map(
             entity_type_handle = H5Writer.fetch_handle(h5file, entity_type)
             map_values = color_map.values
             entity_type_handle.create_dataset(
-                "Color map", map_values.shape, data=map_values, dtype=map_values.dtype
+                "Color map",
+                map_values.shape,
+                data=map_values,
+                dtype=map_values.dtype,
+                compression="gzip",
+                compression_opts=9,
+            )
+
+        if close_file:
+            h5file.close()
+
+    @classmethod
+    def write_value_map(
+        cls,
+        entity_type: "shared.EntityType",
+        file: Optional[Union[str, h5py.File]] = None,
+        close_file: bool = True,
+    ):
+        """
+        Add :obj:`~geoh5py.data.value_map.ColorMap` to a
```

Comment:
**question:** I am confused: is this about color map or reference data?

Reply:
Good eye, typo.